web-admin/README.md

```
@@ -53,36 +53,56 @@ L'API REST se trouve dans le dossier `/api` à la racine du projet et comprend l
-- `POST /api/admin/auth` - Authentification
+#### Authentification
+- `POST /api/admin/auth` - Authentification et génération de token
 - `PUT /api/admin/auth` - Modification de mot de passe
-- `DELETE /api/admin/auth` - Déconnexion
-
-- `GET /api/admin/companies` - Liste des entreprises
-- `GET /api/admin/contracts` - Liste des contrats
-- `GET /api/admin/services` - Liste des services
+- `DELETE /api/admin/auth` - Déconnexion et invalidation du token
+
+#### Gestion des entreprises
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)